scripts/utils.sh

```diff
@@ -16,3 +16,47 @@ RED='\033[0;31m'
 GREEN='\033[0;32m'
 YELLOW='\033[1;33m'
 NC='\033[0m' # No Color
+
+publish_results() {
+  AWS_HOURLY_COST=32.7726
+  AZURE_HOURLY_COST=14.69
+  local step=$1
+  local PROVIDER=$2
+  local result_file="/tmp/${USER_ALIAS}-result.json"
```

Comment:
Let use the same name to distinguish the result from different cloud and for each run_id "${TF_VAR_run_id}-${PROVIDER}.json"